hack/generator/pkg/astmodel/package_definition.go

```diff
@@ -44,13 +44,20 @@ func (pkgDef *PackageDefinition) EmitDefinitions(outputDir string) {
 	allocateTypesToFiles(otherDefinitions, filesToGenerate)
 	emitFiles(filesToGenerate, outputDir)
 	emitGroupVersionFile(pkgDef, outputDir)
+
+	return len(filesToGenerate)
+}
+
+// DefinitionCount returns the count of definitions that have been sorted into this package
+func (pkgDef *PackageDefinition) DefinitionCount() int {
+	return len(pkgDef.definitions)
 }
 
 func emitFiles(filesToGenerate map[string][]Definition, outputDir string) {
 	for fileName, defs := range filesToGenerate {
 		genFile := NewFileDefinition(defs[0].Reference().PackageReference, defs...)
 		outputFile := filepath.Join(outputDir, fileName+"_types.go")
-		log.Printf("Writing '%s'\n", outputFile)
+		klog.V(5).Infof("Writing '%s'\n", outputFile)
```

Comment:
Are there an unbounded number of verbosity levels? How far up do we want to use in code?

Reply:
It goes up to 5 and there are defined uses, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md